app/controllers/application_controller.rb

```diff
@@ -20,8 +20,15 @@ def active_caseload
     sso_identity['active_caseload'] if sso_identity.present?
   end
 
+  def update_active_caseload(code)
+    session[:sso_data]['active_caseload'] = code
+  end
+
   def caseloads
-    sso_identity['caseloads'] if sso_identity.present?
+    return nil if sso_identity.blank?
+
+    caseloads = sso_identity['caseloads']
+    caseloads.reject { |c| c == 'NWEB' }
```

Comment:
How come NWEB get's returned in caseloads?

Reply:
Because NWEB is assigned under the caseload section in NOMIS, the same screen where prisons are added